### PR TITLE
feat(spellcheck): add keyboard shortcut for spellcheck

### PR DESCRIPTION
**Pull request by petrjasek**
it can be triggered via ctrl+shift+s

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/views/authoring-topbar.html [hunk 1]
```diff
@@ -111,7 +111,9 @@
                         <button type="button"
                             ng-click="spellcheckMenu.spellcheck()"
                             ng-disabled="spellcheckMenu.isAuto"
-                            translate>Check spelling</button>
+                            translate>
+                                <span class="shortcut pull-right">Ctrl+Shift+S</span>
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: Should that be "translated" for Mac users somehow?

(meh, screw them :grinning: )

**Comment by plamut** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/views/authoring-topbar.html (hunk 1 above)
Comment (amagdas) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: Hey, I'm using a Mac :)

**Comment by amagdas** [comment]
:+1:

**Pull request merged**